app/views/course/condition/_level.html.slim

```diff
@@ -1,8 +1,8 @@
-= content_tag_for(:tr, level)
+- level_condition = condition.specific
+= content_tag_for(:tr, level_condition, class: :condition)
```

Comment:
it generates `level_condition` class (from the variable name) but I want an extra class shared by all conditions.